__tests__/app.test.js

```diff
@@ -22,133 +20,215 @@ describe('api',()=>{
+                true
+              );
+              expect(
+                category.hasOwnProperty("description", expect.any(String))
+              ).toBe(true);
+            });
+          });
+      });
+    });
+
+    describe("/api/reviews/review_id", () => {
+      it("200 GET - responds with single review object", () => {
+        return request(app)
+          .get("/api/reviews/2")
+          .expect(200)
+          .then(({ body }) => {
+            const review = body.review;
+
+            expect(review.review_id).toBe(2);
+            expect(review.title).toBe("Jenga");
+            expect(review.category).toBe("dexterity");
+            expect(review.designer).toBe("Leslie Scott");
+
+            expect(review.owner).toBe("philippaclaire9");
+            expect(review.review_body).toBe("Fiddly fun for all the family");
+            expect(review.review_img_url).toBe(
+              "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700"
+            );
+            expect(review.created_at).toBe("2021-01-18T10:01:41.251Z");
+            expect(review.votes).toBe(5);
+          });
+      });
+
+      it("400 GET - invalid review id responds with bad request msg.", () => {
+        return request(app)
+          .get("/api/reviews/bad-request")
+          .expect(400)
+          .then(({ body }) => {
+            const serverResponseMsg = body.msg;
+            expect(serverResponseMsg).toBe("Bad request.");
+          });
+      });
+
+      it("404 GET - responds with msg when sent valid but non-existent path", () => {
+        return request(app)
+          .get("/api/reviews/99999999")
+          .expect(404)
+          .then(({ body }) => {
+            const serverResponseMsg = body.msg;
+            expect(serverResponseMsg).toBe("Content not found.");
+          });
+      });
+    });
+
+
+  describe("/api/reviews", () => {
+    it("200 GET - responds array of review objects, including the correct properties ", () => {
       return request(app)
-        .get("/api/categories")
+        .get("/api/reviews")
         .expect(200)
         .then(({ body }) => {
-          const categories = body.categories;
-
-          expect(categories.length).toBe(4);
-          categories.forEach((category) => {
-            expect(category.hasOwnProperty("slug", expect.any(String))).toBe(
+          const reviews = body.reviews;
+          expect(reviews.length).toBe(13);
+          reviews.forEach((review) => {
+            expect(review.hasOwnProperty("owner", expect.any(String))).toBe(
+              true
+            );
+            expect(review.hasOwnProperty("title", expect.any(String))).toBe(
+              true
+            );
+            expect(review.hasOwnProperty("review_id", expect.any(Number))).toBe(
+              true
+            );
+            expect(review.hasOwnProperty("category", expect.any(String))).toBe(
```

Comment:
These tests would be a lot more concise and easier to write with a toMatchObject, though not a dealbreaker